uniswap_swap_bot/src/bot.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+
+import bot from "./bot";
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
```

Comment:
In testing, we typically don't use the _exact_ same variables as those in prod. So I'd recommend declaring local variables inside the test suite file, even if they have the same value.

This is because if a variable is incorrect in the prod, then you use the same variable in testing, your implementation would be consistently incorrect.

For example, you're checking that `filterLog` was called with those values. But if they're incorrect, then you're checking if `filterLog` was called with incorrect values.